inlong-agent/agent-plugins/pom.xml

```diff
@@ -69,6 +69,11 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>org.apache.kafka</groupId>
+            <artifactId>kafka-clients</artifactId>
```

Comment:
why kafka clients here ?

Reply:
Because the debezium-embedded package calls kafka client 2.4.1, but the Utils.isBlank method is connect-runtime 3.0.0. There is a problem with this online logic, it is a bug